array/any.js

```diff
@@ -0,0 +1 @@
+export default xs => xs && xs.length > 0;
```

Comment:
JavaScript already have this as [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some). The `any(xs)` call is equal to C#'s [`xs.Any()` parameterless overload](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.any?view=netframework-4.8#System_Linq_Enumerable_Any__1_System_Collections_Generic_IEnumerable___0__).
The `some`/`every` are commonly used and well supported (IE9+) native Array APIs so IMHO no need to duplicate that.

Reply:
Exactly :) As my first sentence in the comment: I'm thinking about the different name than `any` for that method

Reply:
I try to avoid `empty/isEmpty` to avoid conflict with the `empty` definition (`[]` for array, `{}` for object). However, I consider dropping them to use `empty` as a test function. But there is a naming issue of symmetric function - `nonEmpty`? I would like to avoid joined names and just stick with single-word names where possible. For now, it is `any` and `none`.
One more thing, the functions handle missing argument case, so: any(null) = false, any(undefined) = false, any(0) = false, array.any([]) = false, object.any({}) = false etc.